### PR TITLE
Create 0xsequence.yaml

### PR DESCRIPTION
**Pull request by JamesLawton**

### CODE REVIEW

**Review by ccerv1 — CHANGES_REQUESTED** [review]
This looks great! A few small changes and we can approve:

- [ ] You can remove the factory contract ("0xf9...") since that was deployed by the deployer you've included.
- [ ] For your deployer ("0xB70..."), you can tag the network as `any_evm` instead of `optimism` if you want us to grab any deployment on any EVM chain from this deployer
- [ ] For the deployer, please add an `eoa` tag to it so we treat it as an EOA deployer
- [ ] Check your indentations in the blockchain section. YAML format is fussy about this.

To summarize, I think the updated file should look like this:

```
version: 7
name: 0xsequence
display_name: Sequence
github:
  - url: https://github.com/0xsequence
npm:
  - url: https://www.npmjs.com/package/0xsequence
blockchain:
  - address: "0xB70d2E30B2D2f8af8B657f5Da5B305eC533F6DC2"
    networks:
      - any_evm
    tags:
      - deployer
      - eoa
      - wallet
```

**Review by ccerv1 — APPROVED** [review]
Looks good! Thank you!!! Will validate now

**Review by ccerv1 — APPROVED** [review]
